Other/MergeShops/Adam1a1Merge.cs

```diff
@@ -0,0 +1,239 @@
+                                Core.HuntMonster("Sandsea", "Cactus Creeper", "Fandango Flower", 5);
+                                Core.HuntMonster("Wanders", "Lotus Spider", "Lotus Flower", 4);
+                            }
+
+                            Adv.BuyItem("tower", 347, "Pink Rose");
+                            while(!Bot.ShouldExit && !Core.CheckInventory(21567, 10)){
+                                Core.HuntMonster("battlewedding", "Silver Knight", isTemp: false);
+                            }
+                            Adv.BuyItem("battlewedding", 788, "Red Rose");
+                        }
+
+                        // Stray Ectoplasm
+                        if(!Core.CheckInventory("Stray Ectoplasm")){
+                            while (!Bot.ShouldExit && !Core.CheckInventory("Fresh Ectoplasm", 15))
+                            {   
+                                Core.EnsureAccept(8009);
+                                Core.HuntMonster("vendorbooths", "Caffeine Imp", "Coffee Beans", 10);
+                                Core.HuntMonster("djinn", "Lamia", "Tasty Poison", 10);
+                                Core.HuntMonster("charredpath", "Toxic Wisteria", "Necessary Antidote");
+                                Core.EnsureComplete(8009);
+                                Bot.Wait.ForPickup("Fresh Ectoplasm");
+                            }
+                            Adv.BuyItem("battleunderb", 1990, "Stray Ectoplasm");
+                        }
+                        Core.EnsureComplete(8013);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Reho's Golden Sword Hilt":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    if (!Bot.Quests.IsUnlocked(8011)) {
+                        Adam1a1Quest.doall();
+                    }
+                    Core.RegisterQuests(8011);                    
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("extinction", "Slimed Drone", "Iron II.0", 4, isTemp: false);
+                        Core.HuntMonster("doomwood", "Doomwood Treeant", "Wood", 10);
+                        Core.HuntMonster("crashsite", "Dwakel Blaster", "Big Iron Bolts", 10);
+                        Core.HuntMonster("portalmaze", "Time Wraith", "Piece of Cake", 5);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Awethur's Accoutrements":
+                    Core.FarmingLogger(req.Name, quant);
+                    if (Bot.Reputation.GetRank("Blade of Awe") < 6){
+                        Farm.BladeofAweREP(10, false);
```

Comment:
reps go the line below the stores on L45

all the pre-requisets are to be done before L45